src/sidebar/helpers/export-annotations.ts

```diff
@@ -0,0 +1,32 @@
+import type { Annotation } from '../../types/api';
+
+export type ExportAnnotationsData = {
+  annotations: Annotation[];
+  clientVersion: string;
+  userId: string;
+};
+
+export type ExportContent = {
+  export_date: string;
+  export_userid: string;
+  client_version: string;
+  annotations: Annotation[];
+};
```

Comment:
Right now, both the caller and this helper need to be aware of the "container" metadata for the export file, here typed in `ExportAnnotationsData`. I wonder if it might be possible instead to make this helper take just `annotations` and do the container-property building itself. It could simplify the API and typing, because right now `ExportAnnotationsData` and `ExportContent` pertain to overlapping properties (`clientVersion` => `client_version`, e.g.) and this could be confusing over time, especially if the amount of container properties gets long.

To accomplish this, it's possible this might have to be a service instead of a helper, but I think its worth considering.

Reply:
That actually makes a lot of sense.